console/console-time-label-conversion.any.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+// https://console.spec.whatwg.org/#timing
+
+test(() => {
+  let timeLabelToStringCalled = false;
+  let timeEndLabelToStringCalled = false;
+
+  console.time({
+    toString() {
+      timeLabelToStringCalled = true;
+    }
+  });
+
+  console.timeEnd({
+    toString() {
+      timeEndLabelToStringCalled = true;
+    }
+  });
+
+  assert_true(timeLabelToStringCalled, "toString() was never called on time()'s label when label is an object");
```

Comment:
It's best to state the assertions as things that should be true, e.g. "toString() must be called on time()'s label when label is an object"